docs/content/best-practices/immutable-fields.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Immutable fields"
+weight: 10
```

Comment:
We should preserve the old redirect & add a new one. I'd suggest here because it's the first new page.

```suggestion
weight: 10
aliases:
  - /docs/best-practices
  - /best-practices
```

Reply:
Oh, I'd tried just /docs/best-practices and it didn't seem to work so I removed it. That explains it!